src/App.vue

```diff
@@ -1,15 +1,573 @@
 <template>
-  <img alt="Vue logo" src="./assets/logo.png">
-  <HelloWorld msg="Welcome to Your Vue.js App"/>
+<Hero />
+<WhatIsSus />
+<EventSchedule @transfer-click="transferClickNum" @transfer-date="transferDate"/>
+
+<!--Cards for First day -->
+<div class="card_container" v-if="choice === 1">
```

Comment:
Can we move these details into a different vue file as a component or as a page?